packages/kirakira-backend/DefaultApi.md

```diff
@@ -340,6 +342,60 @@ No authorization required
 
 [[Back to top]](#) [[Back to API list]](README.md#documentation-for-api-endpoints) [[Back to Model list]](README.md#documentation-for-models) [[Back to README]](README.md)
 
+# **recommendations**
+> Array<Videos200ResponseVideosInner> recommendations()
+
+
+### Example
+
+
+```typescript
+import {  } from '';
```

Comment:
?.?

Reply:
lol wtf happened there

Reply:
oh but that's generated code, I didn't write it. IDK TBH